lib/view_component/form/builder.rb

```diff
@@ -15,6 +15,7 @@ class << self
 
       self.components_namespace = "ViewComponent::Form"
 
+      # TODO: remove file_field?
       (field_helpers - %i[label check_box radio_button fields_for fields hidden_field file_field]).each do |selector|
```

Comment:
@nicolas-brousse do you know why `file_field` is removed from the list of `field_helpers` here?

Reply:
I guess it's because it needs to set multipart to `true`.

https://github.com/rails/rails/blob/83217025a171593547d1268651b446d3533e2019/actionview/lib/action_view/helpers/form_helper.rb#L2417